cmd/template.go

```diff
@@ -55,14 +59,30 @@ func template(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	append, err := cmd.Flags().GetStringArray(templateCommandAppendArgument)
+	if err != nil {
+		return err
+	}
+
+	appendMap = make(map[string]string)
+	for _, val := range append {
+		parts := strings.Split(val, ":")
```

Comment:
This part will fail if val will have multiple `:`. See https://github.com/AirHelp/treasury/blob/master/utils/utils.go#L62 for inspiration :trollface: